### PR TITLE
doc: jan-2022 sec release post-announcement

### PR DESCRIPTION
**Pull request by bengl**

### CODE REVIEW

**Review by tniessen — APPROVED** [review]

**Review by BethGriggs — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

File: locale/en/blog/vulnerability/jan-2022-security-releases.md [hunk 1]
```diff
@@ -1,12 +1,82 @@
 ---
-date: 2022-01-04T16:00:00.000Z
+date: 2022-01-10T16:00:00.000Z
```
Comment (richardlau) [inline locale/en/blog/vulnerability/jan-2022-security-releases.md]: We may need to move this forward if we want this to appear above the release posts?

**Review by richardlau — APPROVED** [review]

File: locale/en/blog/vulnerability/jan-2022-security-releases.md (hunk 1 above)
Comment (mhdawson) [reply locale/en/blog/vulnerability/jan-2022-security-releases.md]: +1, We should move it to be right now

Comment (mhdawson) [reply locale/en/blog/vulnerability/jan-2022-security-releases.md]: 2022-01-11T00:48:00:00.000Z would probably make sense.

Comment (tniessen) [inline locale/en/blog/vulnerability/jan-2022-security-releases.md]:
```suggestion
date: 2022-01-11T00:50:00.000Z
```

Comment (mhdawson) [reply locale/en/blog/vulnerability/jan-2022-security-releases.md]: I think it should be a bit later

date: 2022-01-11T50:00:00.000Z

Comment (mhdawson) [reply locale/en/blog/vulnerability/jan-2022-security-releases.md]: no that's wrong

I mean

date: 2022-01-11T00:50:00.000Z

Comment (mhdawson) [reply locale/en/blog/vulnerability/jan-2022-security-releases.md]: Looks good to me now

**Review by richardlau — APPROVED** [review]

**Pull request merged**